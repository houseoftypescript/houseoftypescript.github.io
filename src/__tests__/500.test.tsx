import { render } from '@testing-library/react';
import InternalServerErrorPage from '../pages';

describe('InternalServerErrorPage', () => {
  it('renders correctly', () => {
    const button = render(<InternalServerErrorPage />);
    expect(button.container).toMatchSnapshot();
  });
});
