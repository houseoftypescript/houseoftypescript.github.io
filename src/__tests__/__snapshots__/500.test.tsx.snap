// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InternalServerErrorPage renders correctly 1`] = `
<div>
  <div
    class="sticky-outer-wrapper"
  >
    <div
      class="sticky-inner-wrapper"
      style="position: relative; z-index: 100; transform: translate3d(0,0px,0);"
    >
      <nav
        class="bg-white"
      >
        <div
          class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
        >
          <div
            class="py-8"
          >
            <div
              class="flex justify-between items-center"
            >
              <div
                class="flex items-center gap-8"
              >
                <h1
                  class="text-2xl"
                >
                  <a
                    class="cursor-pointer"
                  >
                    HOM
                  </a>
                </h1>
                <div
                  class="hidden md:flex items-center gap-8"
                >
                  <div>
                    <a
                      class="cursor-pointer nav-scroll-link"
                    >
                      Home
                    </a>
                  </div>
                  <div>
                    <a
                      class="cursor-pointer nav-scroll-link"
                    >
                      Services
                    </a>
                  </div>
                  <div>
                    <a
                      class="cursor-pointer nav-scroll-link"
                    >
                      Team
                    </a>
                  </div>
                  <div>
                    <a
                      class="cursor-pointer nav-scroll-link"
                    >
                      Why Us
                    </a>
                  </div>
                  <div>
                    <a
                      class="cursor-pointer nav-scroll-link"
                    >
                      Projects
                    </a>
                  </div>
                </div>
              </div>
              <div
                class="flex items-center gap-8"
              >
                <a
                  class="cursor-pointer text-emerald-500 font-bold"
                >
                  Get Started
                </a>
                <div
                  class="block md:hidden"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                    data-testid="MenuIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
                    />
                  </svg>
                </div>
              </div>
            </div>
          </div>
        </div>
      </nav>
    </div>
  </div>
  <main>
    <section
      class="py-8 md:py-16"
      id="home"
    >
      <div
        class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
      >
        <div
          class="mb-8 md:mb-16"
        >
          <div
            class="text-center max-w-lg mx-auto"
          >
            <h1
              class="text-2xl sm:text-4xl mb-4 font-b162 font-bold"
            >
              Creativity starts with viewing the whole world differently
            </h1>
            <h2
              class="text-sm sm:text-lg"
            >
              A service agency with customer satisfaction
            </h2>
          </div>
        </div>
        <div
          class="relative"
        >
          <div
            class="sm:absolute top-0 w-full flex justify-center pb-16"
          >
            <a
              class="cursor-pointer"
            >
              <button
                class="px-8 py-4 rounded-md bg-emerald-500 hover:bg-gray-900 text-white transition duration-500 "
                type="button"
              >
                Get Started
              </button>
            </a>
          </div>
          <img
            alt="Banner"
            class="hidden sm:block"
            data-nimg="1"
            decoding="async"
            height="40"
            loading="lazy"
            src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
            srcset="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
            style="color: transparent;"
            width="40"
          />
        </div>
      </div>
      <div
        class="relative hidden sm:block"
      >
        <div
          class="bottom-0 -z-10 absolute bg-[#FFF5ED]/50 w-full h-32"
        />
      </div>
    </section>
    <section
      class="bg-[#FFF5ED]/50 py-8 md:py-16"
      id="services"
    >
      <div
        class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
      >
        <div
          class="mb-8 md:mb-16"
        >
          <div
            class="text-center max-w-lg mx-auto"
          >
            <h1
              class="text-xl sm:text-3xl mb-4 font-b162 font-bold"
            >
              Grow your startup with our Service
            </h1>
            <h2
              class="text-sm sm:text-lg"
            >
              Build an incredible workplace and grow your business with Gustoâ€™s all-in-one platform with amazing contents.
            </h2>
          </div>
        </div>
        <div
          class="grid grid-cols-1 sm:grid-cols-3 gap-8"
        >
          <div
            class="grid grid-cols-1 sm:grid-cols-3 gap-x-0 gap-y-8 md:gap-8"
          >
            <div
              class="col-span-1"
            >
              <img
                alt="Ultimate development"
                class="w-16 h-16 mx-auto"
                data-nimg="1"
                decoding="async"
                height="40"
                loading="lazy"
                src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
                srcset="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
                style="color: transparent;"
                width="40"
              />
            </div>
            <div
              class="col-span-2"
            >
              <div
                class="text-center sm:text-left"
              >
                <div
                  class="mb-4"
                >
                  <h3
                    class="font-bold text-md sm:text-lg"
                  >
                    Ultimate development
                  </h3>
                </div>
                <p>
                  Get your info tests delivered at home collect a sample from the your progress tests.
                </p>
              </div>
            </div>
          </div>
          <div
            class="grid grid-cols-1 sm:grid-cols-3 gap-x-0 gap-y-8 md:gap-8"
          >
            <div
              class="col-span-1"
            >
              <img
                alt="TypeScript"
                class="w-16 h-16 mx-auto"
                data-nimg="1"
                decoding="async"
                height="40"
                loading="lazy"
                src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
                srcset="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
                style="color: transparent;"
                width="40"
              />
            </div>
            <div
              class="col-span-2"
            >
              <div
                class="text-center sm:text-left"
              >
                <div
                  class="mb-4"
                >
                  <h3
                    class="font-bold text-md sm:text-lg"
                  >
                    TypeScript
                  </h3>
                </div>
                <p>
                  JavaScript superset with optional static typing for improved code quality.
                </p>
              </div>
            </div>
          </div>
          <div
            class="grid grid-cols-1 sm:grid-cols-3 gap-x-0 gap-y-8 md:gap-8"
          >
            <div
              class="col-span-1"
            >
              <img
                alt="Node.js"
                class="w-16 h-16 mx-auto"
                data-nimg="1"
                decoding="async"
                height="40"
                loading="lazy"
                src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
                srcset="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
                style="color: transparent;"
                width="40"
              />
            </div>
            <div
              class="col-span-2"
            >
              <div
                class="text-center sm:text-left"
              >
                <div
                  class="mb-4"
                >
                  <h3
                    class="font-bold text-md sm:text-lg"
                  >
                    Node.js
                  </h3>
                </div>
                <p>
                  JavaScript runtime for building server-side and network apps.
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section
      class="bg-[#FFF5ED]/50 py-8 md:py-16 testimonials-container"
    >
      <div
        class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
      >
        <div
          class="rc-tabs rc-tabs-bottom css-0"
        >
          <div
            class="rc-tabs-nav"
            role="tablist"
          >
            <div
              class="rc-tabs-nav-wrap"
            >
              <div
                class="rc-tabs-nav-list"
                style="transform: translate(0px, 0px);"
              >
                <div
                  class="rc-tabs-ink-bar rc-tabs-ink-bar-animated"
                />
              </div>
            </div>
            <div
              class="rc-tabs-nav-operations rc-tabs-nav-operations-hidden"
            >
              <button
                aria-controls="rc-tabs-test-more-popup"
                aria-expanded="false"
                aria-haspopup="listbox"
                aria-hidden="true"
                class="rc-tabs-nav-more"
                id="rc-tabs-test-more"
                style="visibility: hidden; order: 1;"
                tabindex="-1"
                type="button"
              >
                More
              </button>
            </div>
          </div>
          <div
            class="rc-tabs-content-holder"
          >
            <div
              class="rc-tabs-content rc-tabs-content-bottom"
            />
          </div>
        </div>
      </div>
    </section>
    <section
      class="py-8 md:py-16"
      id="team"
    >
      <div
        class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
      >
        <div
          class="mb-16"
        >
          <div
            class="mb-8 md:mb-16"
          >
            <div
              class="text-center max-w-lg mx-auto"
            >
              <h1
                class="text-xl sm:text-3xl mb-4 font-b162 font-bold"
              >
                Meet our superheros
              </h1>
              <h2
                class="text-sm sm:text-lg"
              >
                Our team of software engineers, and business analysts with expertise in mobile, web, cloud technologies to create custom solutions that cater to your specific business needs.
              </h2>
            </div>
          </div>
        </div>
        <div
          class="grid grid-col-1 md:grid-cols-3 gap-8"
        >
          <div
            class="col-span-1"
          >
            <div>
              <img
                alt="Hieu Doan"
                class="rounded-xl border"
                data-nimg="1"
                decoding="async"
                height="40"
                loading="lazy"
                src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
                srcset="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
                style="color: transparent;"
                width="40"
              />
              <div
                class="text-center"
              >
                <h2
                  class="mt-4 text-lg md:text-2xl font-medium"
                >
                  Hieu Doan
                </h2>
                <p
                  class="mt-2 text-sm md:text-lg text-gray-500"
                >
                  Full-stack Engineer
                </p>
              </div>
            </div>
          </div>
          <div
            class="col-span-1"
          >
            <div>
              <img
                alt="Phuong Tran"
                class="rounded-xl border"
                data-nimg="1"
                decoding="async"
                height="40"
                loading="lazy"
                src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
                srcset="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
                style="color: transparent;"
                width="40"
              />
              <div
                class="text-center"
              >
                <h2
                  class="mt-4 text-lg md:text-2xl font-medium"
                >
                  Phuong Tran
                </h2>
                <p
                  class="mt-2 text-sm md:text-lg text-gray-500"
                >
                  Business Analysts
                </p>
              </div>
            </div>
          </div>
          <div
            class="col-span-1"
          >
            <div>
              <img
                alt="Duy Ton"
                class="rounded-xl border"
                data-nimg="1"
                decoding="async"
                height="40"
                loading="lazy"
                src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
                srcset="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
                style="color: transparent;"
                width="40"
              />
              <div
                class="text-center"
              >
                <h2
                  class="mt-4 text-lg md:text-2xl font-medium"
                >
                  Duy Ton
                </h2>
                <p
                  class="mt-2 text-sm md:text-lg text-gray-500"
                >
                  Data Engineer
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section
      class="py-8 md:py-16 bg-[#F9FAFC]"
    >
      <div
        class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
      >
        <div
          class="mb-8 md:mb-16"
        >
          <div
            class="text-center max-w-lg mx-auto"
          >
            <h1
              class="text-xl sm:text-3xl mb-4 font-b162 font-bold"
            >
              Our modern web techstack
            </h1>
            <h2
              class="text-sm sm:text-lg"
            >
              Focus only on the meaning, we take care of the design. As soon as the meeting end you can export in one click into your preferred.
            </h2>
          </div>
        </div>
        <div
          class="grid grid-cols-1 sm:grid-cols-2 md:grid-cols-3 gap-8"
        >
          <div
            class="grid grid-cols-1 sm:grid-cols-3 gap-x-0 gap-y-8 md:gap-8"
          >
            <div
              class="col-span-1"
            >
              <img
                alt="Next.js"
                class="w-16 h-16 mx-auto"
                data-nimg="1"
                decoding="async"
                height="40"
                loading="lazy"
                src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
                srcset="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
                style="color: transparent;"
                width="40"
              />
            </div>
            <div
              class="col-span-2"
            >
              <div
                class="text-center sm:text-left"
              >
                <div
                  class="mb-4"
                >
                  <h3
                    class="font-bold text-md sm:text-lg"
                  >
                    Next.js
                  </h3>
                </div>
                <p>
                  React framework for server-rendered web apps with powerful features.
                </p>
              </div>
            </div>
          </div>
          <div
            class="grid grid-cols-1 sm:grid-cols-3 gap-x-0 gap-y-8 md:gap-8"
          >
            <div
              class="col-span-1"
            >
              <img
                alt="MUI"
                class="w-16 h-16 mx-auto"
                data-nimg="1"
                decoding="async"
                height="40"
                loading="lazy"
                src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
                srcset="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
                style="color: transparent;"
                width="40"
              />
            </div>
            <div
              class="col-span-2"
            >
              <div
                class="text-center sm:text-left"
              >
                <div
                  class="mb-4"
                >
                  <h3
                    class="font-bold text-md sm:text-lg"
                  >
                    MUI
                  </h3>
                </div>
                <p>
                  Popular React library for Material Design components and theming.
                </p>
              </div>
            </div>
          </div>
          <div
            class="grid grid-cols-1 sm:grid-cols-3 gap-x-0 gap-y-8 md:gap-8"
          >
            <div
              class="col-span-1"
            >
              <img
                alt="GraphQL"
                class="w-16 h-16 mx-auto"
                data-nimg="1"
                decoding="async"
                height="40"
                loading="lazy"
                src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
                srcset="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
                style="color: transparent;"
                width="40"
              />
            </div>
            <div
              class="col-span-2"
            >
              <div
                class="text-center sm:text-left"
              >
                <div
                  class="mb-4"
                >
                  <h3
                    class="font-bold text-md sm:text-lg"
                  >
                    GraphQL
                  </h3>
                </div>
                <p>
                  Flexible query language for API's; client defines data structure needed.
                </p>
              </div>
            </div>
          </div>
          <div
            class="grid grid-cols-1 sm:grid-cols-3 gap-x-0 gap-y-8 md:gap-8"
          >
            <div
              class="col-span-1"
            >
              <img
                alt="PostgreSQL"
                class="w-16 h-16 mx-auto"
                data-nimg="1"
                decoding="async"
                height="40"
                loading="lazy"
                src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
                srcset="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
                style="color: transparent;"
                width="40"
              />
            </div>
            <div
              class="col-span-2"
            >
              <div
                class="text-center sm:text-left"
              >
                <div
                  class="mb-4"
                >
                  <h3
                    class="font-bold text-md sm:text-lg"
                  >
                    PostgreSQL
                  </h3>
                </div>
                <p>
                  Powerful, open-source, object-relational database management system.
                </p>
              </div>
            </div>
          </div>
          <div
            class="grid grid-cols-1 sm:grid-cols-3 gap-x-0 gap-y-8 md:gap-8"
          >
            <div
              class="col-span-1"
            >
              <img
                alt="Prisma"
                class="w-16 h-16 mx-auto"
                data-nimg="1"
                decoding="async"
                height="40"
                loading="lazy"
                src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
                srcset="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
                style="color: transparent;"
                width="40"
              />
            </div>
            <div
              class="col-span-2"
            >
              <div
                class="text-center sm:text-left"
              >
                <div
                  class="mb-4"
                >
                  <h3
                    class="font-bold text-md sm:text-lg"
                  >
                    Prisma
                  </h3>
                </div>
                <p>
                  Open-source, type-safe ORM for Node.js and TypeScript with GraphQL support.
                </p>
              </div>
            </div>
          </div>
          <div
            class="grid grid-cols-1 sm:grid-cols-3 gap-x-0 gap-y-8 md:gap-8"
          >
            <div
              class="col-span-1"
            >
              <img
                alt="Supabase"
                class="w-16 h-16 mx-auto"
                data-nimg="1"
                decoding="async"
                height="40"
                loading="lazy"
                src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
                srcset="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
                style="color: transparent;"
                width="40"
              />
            </div>
            <div
              class="col-span-2"
            >
              <div
                class="text-center sm:text-left"
              >
                <div
                  class="mb-4"
                >
                  <h3
                    class="font-bold text-md sm:text-lg"
                  >
                    Supabase
                  </h3>
                </div>
                <p>
                  Open-source, Firebase-like platform built on PostgreSQL and GraphQL.
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section
      class="py-8 md:py-16 why-container"
      id="why"
    >
      <div
        class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
      >
        <div
          class="rc-tabs rc-tabs-top css-0"
        >
          <div
            class="rc-tabs-nav"
            role="tablist"
          >
            <div
              class="rc-tabs-nav-wrap"
            >
              <div
                class="rc-tabs-nav-list"
                style="transform: translate(0px, 0px);"
              >
                <div
                  class="rc-tabs-ink-bar rc-tabs-ink-bar-animated"
                />
              </div>
            </div>
            <div
              class="rc-tabs-nav-operations rc-tabs-nav-operations-hidden"
            >
              <button
                aria-controls="rc-tabs-test-more-popup"
                aria-expanded="false"
                aria-haspopup="listbox"
                aria-hidden="true"
                class="rc-tabs-nav-more"
                id="rc-tabs-test-more"
                style="visibility: hidden; order: 1;"
                tabindex="-1"
                type="button"
              >
                More
              </button>
            </div>
          </div>
          <div
            class="rc-tabs-content-holder"
          >
            <div
              class="rc-tabs-content rc-tabs-content-top"
            />
          </div>
        </div>
      </div>
    </section>
    <section
      class="py-8 md:py-16"
      id="projects"
    >
      <div
        class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
      >
        <div
          class="mb-8 md:mb-16"
        >
          <div
            class="text-center max-w-lg mx-auto"
          >
            <h1
              class="text-xl sm:text-3xl mb-4 font-b162 font-bold"
            >
              Portfolio of Our Demo Project
            </h1>
            <h2
              class="text-sm sm:text-lg"
            >
              Focus only on the meaning, we take care of the design. As soon as the meeting end you can export in one click.
            </h2>
          </div>
        </div>
        <div
          class="grid grid-cols-1 md:grid-cols-3 gap-8"
        >
          <div
            class="hover:p-4 hover:shadow hover:border transition-all duration-100"
          >
            <a
              class="block"
              href="/themes/next-profile-astronout"
              target="_blank"
            >
              <div
                class="aspect-video rounded border bg-cover bg-center"
                style="background-image: url(/img.jpg);"
              />
              <h2
                class="text-lg font-bold mt-4"
              >
                How to work with prototype design with adobe xd featuring tools
              </h2>
            </a>
            <p
              class="text-gray-500 mt-4"
            >
              The 2019 Innovation by Design Awards honor the designers and businesses solving the problems of today and tomorrow. It is one of the most sought-after design
            </p>
            <div
              class="mt-4"
            >
              <div
                class="MuiChip-root MuiChip-filled MuiChip-sizeSmall MuiChip-colorDefault MuiChip-filledDefault text-white bg-emerald-500 css-1fj5pxn-MuiChip-root"
              >
                <span
                  class="MuiChip-label MuiChip-labelSmall css-wjsjww-MuiChip-label"
                >
                  next.js
                </span>
              </div>
            </div>
          </div>
          <div
            class="hover:p-4 hover:shadow hover:border transition-all duration-100"
          >
            <a
              class="block"
              href="#how-to"
              target="_blank"
            >
              <div
                class="aspect-video rounded border bg-cover bg-center"
                style="background-image: url(/img.jpg);"
              />
              <h2
                class="text-lg font-bold mt-4"
              >
                Anti bias receives honorable gift mention at Fast Company's most Innovation by Design Awards
              </h2>
            </a>
            <p
              class="text-gray-500 mt-4"
            >
              The 2019 Innovation by Design Awards honor the designers and businesses solving the problems of today and tomorrow. It is one of the most sought-after design
            </p>
            <div
              class="mt-4"
            />
          </div>
          <div
            class="hover:p-4 hover:shadow hover:border transition-all duration-100"
          >
            <a
              class="block"
              href="#how-to"
              target="_blank"
            >
              <div
                class="aspect-video rounded border bg-cover bg-center"
                style="background-image: url(/img.jpg);"
              />
              <h2
                class="text-lg font-bold mt-4"
              >
                Multiple task wireframing with team management perform better
              </h2>
            </a>
            <p
              class="text-gray-500 mt-4"
            >
              The 2019 Innovation by Design Awards honor the designers and businesses solving the problems of today and tomorrow. It is one of the most sought-after design
            </p>
            <div
              class="mt-4"
            />
          </div>
        </div>
      </div>
    </section>
    <section
      class="py-0 md:py-16 bg-[#F8F0EA] bg-bottom bg-contain bg-no-repeat"
      id="contact"
      style="background-image: url(/img.jpg);"
    >
      <div
        class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
      >
        <div
          class="bg-white rounded-none md:rounded-2xl"
        >
          <div
            class="px-4 py-8 md:px-8 md:py-16"
          >
            <div
              class="grid grid-cols-1 md:grid-cols-2 items-center gap-8"
            >
              <div
                class="col-span-1"
              >
                <div
                  class=""
                >
                  <div
                    class="text-center md:text-left max-w-lg mx-auto"
                  >
                    <h1
                      class="text-2xl sm:text-4xl mb-4 font-b162 font-bold"
                    >
                      Like our service? Contact us
                    </h1>
                    <h2
                      class="text-sm sm:text-lg"
                    >
                      Looking for a reliable software development company? Contact us and we will get back to you shortly!
                    </h2>
                  </div>
                </div>
              </div>
              <div
                class="col-span-1"
              >
                <div
                  class="grid grid-cols-1 md:grid-cols-3 gap-4"
                >
                  <div
                    class="col-span-1 md:col-span-2"
                  >
                    <input
                      class="w-full px-8 py-4 rounded-md bg-[#EFF3F7] hover:border-emerald-500 active:border-emerald-500 focus:border-emerald-500"
                      id="email"
                      name="email"
                      placeholder="Enter Email Address"
                      type="text"
                    />
                  </div>
                  <div
                    class="col-span-1"
                  >
                    <button
                      class="px-8 py-4 rounded-md bg-emerald-500 hover:bg-gray-900 text-white transition duration-500 w-full"
                      type="button"
                    >
                      Send
                    </button>
                  </div>
                  <div
                    class="col-span-1 md:col-span-3"
                  >
                    <label
                      class="flex items-center gap-2"
                      for="promote"
                    >
                      <input
                        class="hidden"
                        id="promote"
                        name=""
                        type="checkbox"
                        value=""
                      />
                      <div
                        class="w-5 h-5 rounded border border-2"
                      />
                      <span
                        class="text-sm text-gray-500"
                      >
                        Don't provide any promotional message.
                      </span>
                    </label>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
  </main>
  <footer
    class="py-8 border-t md:border-none"
  >
    <div
      class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
    >
      <div
        class="flex items-center justify-between"
      >
        <div
          class="font-medium font-lg"
        >
          Â© 
          2023
           
          HOM
        </div>
        <div
          class="flex items-center gap-2 md:gap-4"
        >
          <a
            href="https://facebook.com"
            target="_blank"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium text-[#1877f2] css-i4bv87-MuiSvgIcon-root"
              data-testid="FacebookIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M5 3h14a2 2 0 0 1 2 2v14a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2m13 2h-2.5A3.5 3.5 0 0 0 12 8.5V11h-2v3h2v7h3v-7h3v-3h-3V9a1 1 0 0 1 1-1h2V5z"
              />
            </svg>
          </a>
          <a
            href="https://twitter.com"
            target="_blank"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium text-[#1da1f2] css-i4bv87-MuiSvgIcon-root"
              data-testid="TwitterIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M22.46 6c-.77.35-1.6.58-2.46.69.88-.53 1.56-1.37 1.88-2.38-.83.5-1.75.85-2.72 1.05C18.37 4.5 17.26 4 16 4c-2.35 0-4.27 1.92-4.27 4.29 0 .34.04.67.11.98C8.28 9.09 5.11 7.38 3 4.79c-.37.63-.58 1.37-.58 2.15 0 1.49.75 2.81 1.91 3.56-.71 0-1.37-.2-1.95-.5v.03c0 2.08 1.48 3.82 3.44 4.21a4.22 4.22 0 0 1-1.93.07 4.28 4.28 0 0 0 4 2.98 8.521 8.521 0 0 1-5.33 1.84c-.34 0-.68-.02-1.02-.06C3.44 20.29 5.7 21 8.12 21 16 21 20.33 14.46 20.33 8.79c0-.19 0-.37-.01-.56.84-.6 1.56-1.36 2.14-2.23z"
              />
            </svg>
          </a>
          <a
            href="https://github.com"
            target="_blank"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium text-[#333333] css-i4bv87-MuiSvgIcon-root"
              data-testid="GitHubIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M12 1.27a11 11 0 00-3.48 21.46c.55.09.73-.28.73-.55v-1.84c-3.03.64-3.67-1.46-3.67-1.46-.55-1.29-1.28-1.65-1.28-1.65-.92-.65.1-.65.1-.65 1.1 0 1.73 1.1 1.73 1.1.92 1.65 2.57 1.2 3.21.92a2 2 0 01.64-1.47c-2.47-.27-5.04-1.19-5.04-5.5 0-1.1.46-2.1 1.2-2.84a3.76 3.76 0 010-2.93s.91-.28 3.11 1.1c1.8-.49 3.7-.49 5.5 0 2.1-1.38 3.02-1.1 3.02-1.1a3.76 3.76 0 010 2.93c.83.74 1.2 1.74 1.2 2.94 0 4.21-2.57 5.13-5.04 5.4.45.37.82.92.82 2.02v3.03c0 .27.1.64.73.55A11 11 0 0012 1.27"
              />
            </svg>
          </a>
        </div>
      </div>
    </div>
  </footer>
</div>
`;
